controllers/sr_controller/sr_controller.py

```diff
@@ -134,9 +134,14 @@ def reconfigure_environment(robot_file: Path) -> None:
     """
 
     # Remove ourselves from the path and insert the competitor code
-    sys.path.pop()
-    sys.path.insert(0, str(ROOT / "modules"))
-    sys.path.insert(0, str(robot_file.parent))
+    sys.path[:1] = [
```

Comment:
Remind me why we need to remove ourselves from the path?

Reply:
Only so that they can't import the controller. I suppose it probably doesn't matter much if they can.

Reply:
I don't think we really need to worry about that. Keeping it as `sys.path.insert` is a lot more readable and approachable.

Reply:
3fec9b2 switches back to the simpler spelling.